CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### 0.10.5 (Next)
 
-* Your contribution here.
+-* Your contribution here.
```

Comment:
This is still busted.

![screen shot 2016-01-06 at 4 25 54 pm](https://cloud.githubusercontent.com/assets/542335/12155237/2f363fe6-b492-11e5-8e76-9c03bc90126e.png)